.github/workflows/triage.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow is built to manage the triage support by using GH issues.
+name: '[Support] Organize triage'
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+  pull_request_target:
+    types:
+      - reopened
+      - opened
+permissions:
+  repository-projects: write
+  issues: write
+
+# To fix the concurrency when for example more than one label is added
+concurrency:
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
```

Comment:
I think this section is not needed. We have only one job and this group is taking into account only `run_id` which is unique for a particular execution, so with this block or without it the workflow can run in parallel if several issues/PR are opened or reopened  at the same time

Reply:
Good catch :)